spring-cloud/spring-cloud-kubernetes-2/pom.xml

```
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.spring.cloud</groupId>
+    <artifactId>spring-cloud-kubernetes-2</artifactId>
+    <version>1.0-SNAPSHOT</version>
+	<name>spring-cloud-kubernetes-2</name>
```

change tab to spaces
```suggestion
    <name>spring-cloud-kubernetes-2</name>
```